tdrs-backend/docs/user_role_management.md

```diff
@@ -0,0 +1,93 @@
+# Updating User Roles through the CLI
+
+**Description**
+For the OTA MVP, we will need to assign user roles via the Python CLI to the deployed applicataion.
```

Comment:
Is this documentation up to date with the latest reflected in the PR?

Reply:
That makes sense - updating with the Django Admin info

Reply:
@alexsoble This has been updated

Reply:
👍 👍